.ci/jsoref-spellchecker/whitelist.words

```diff
@@ -330,19 +331,22 @@ dfn
 Dfoo
 Dforbiddenapis
 Dgpg
+Dgroup
 Dhttp
 Diachenko
 dirname
 Djacoco
 Dlinkcheck
 DMail
 Dmaven
+Dnew
 doccheck
 DOCKERHUB
 Dockter
 doclet
 DOCTYPE
 documentationcomments
+Dold
```

Comment:
Remove all additions to this file that are not hard requirements

Reply:
Changes Done